src/SFA.DAS.ApprenticeFeedback.Data.UnitTests/DatabaseMock/TestAsyncEnumerableEfCore.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Microsoft.EntityFrameworkCore.Query.Internal;
+﻿using Microsoft.EntityFrameworkCore.Query;
```

Comment:
The `IAsyncQueryProvider` referenced in this file no longer lives in the `...Internal` namespace. Even on .NET Core 3.1 this came with a big Intellisense warning, and we should never have been using it.